src/ol/renderer/maprenderer.js

```diff
@@ -171,6 +171,26 @@ ol.renderer.Map.prototype.forEachFeatureAtPixel =
 
 
 /**
+ * @param {ol.Coordinate} coordinate Coordinate.
+ * @param {olx.FrameState} frameState FrameState.
+ * @param {function(this: U, ol.layer.Layer): boolean} layerFilter Layer filter
+ *     function, only layers which are visible and for which this function
+ *     returns `true` will be tested for features.  By default, all visible
+ *     layers will be tested.
+ * @param {U} thisArg Value to use as `this` when executing `layerFilter`.
+ * @return {boolean} Is there a feature at the given pixel?
+ * @template U
+ */
+ol.renderer.Map.prototype.hasFeatureAtPixel =
+    function(coordinate, frameState, layerFilter, thisArg) {
+  var hasFeature = this.forEachFeatureAtPixel(
+      coordinate, frameState, goog.functions.TRUE, this, layerFilter, thisArg);
```

Comment:
> I'm confused why this wouldn't call hasFeatureAtPixel for each layer renderer (with a visible layer).

This one is the default implementation, used by the Canvas and DOM renderers. It is based on the `forEachFeatureAtPixel` function.

The WebGL implementation does call `hasFeatuteAtPixel` for each layer renderer.